package.json

```diff
@@ -293,6 +293,11 @@
         "title": "Toggle follow cursor",
         "category": "LaTeX Workshop"
       },
+      {
+        "command": "latex-workshop.paste",
```

Comment:
Can we merge the two pastes together as one command? I don't think overwriting `ctrl+v` shortcut is a good idea. We came across similar things previously (the enter feature), and a huge number of side-effects need to be resolved.

Reply:
Didn't think about that enter issue. Done in 4af193574c0483b73c57ede6387cd92e092bbecf